traversal/procedure/ProcedureEdge.java

```diff
@@ -827,16 +834,16 @@ public abstract Forwardable<? extends ThingVertex, Order.Asc> branch(
                     GraphManager graphMgr, Vertex<?, ?> fromVertex, Traversal.Parameters params
             );
 
-            Forwardable<ThingVertex, Order.Asc> backwardBranchToIIDFiltered(
```

Comment:
no point creating a more complex `Iterator` object if we know an IID will only match up to 1 vertex!